pkg/agent/controller/networkpolicy/priority.go

```diff
@@ -25,14 +25,18 @@ import (
 )
 
 const (
-	MaxUint16                = ^uint16(0)
-	PolicyBottomPriority     = uint16(100)
-	PolicyTopPriority        = uint16(65000)
-	zoneOffset               = uint16(5)
-	PriorityOffsetSingleTier = float64(640)
-	TierOffsetSingleTier     = uint16(0)
-	PriorityOffsetMultiTier  = float64(20)
-	TierOffsetMultiTier      = uint16(250)
+	MaxUint16                    = ^uint16(0)
```

Comment:
Could we document the impact of these limits in networkpolicy.md?

Reply:
Absolutely. I was thinking on adding this in another PR, which does all the documentations on the baseline tier, pipeline tables changes etc. after we close all comments on this PR.